packages/wdio-allure-reporter/src/index.ts

```diff
@@ -126,6 +127,11 @@ class AllureReporter extends WDIOReporter {
     }
 
     onTestStart(test: TestStats | HookStats) {
+        let testObj:any =test
+        let dataTable = testObj.argument ? testObj.argument.rows.map((a:{cells:null}) => a.cells) : undefined
+        dataTable ? stringify(dataTable, function (err, records) {
+            records ? AllureReporter.addAttachment('Data Table', records, 'text/csv') : console.log(err)
```

Comment:
please don't use static methods of reporter instead use https://github.com/webdriverio/webdriverio/pull/7081/files#diff-c5907206259e7237a9d12ae4b8beff6bd37b92840964c685d194e4cc3233cd75R450

Reply:
@BorisOsipov  instead use ? it points to this diff itself . Could you cross check the link

Reply:
@praveendvd sure https://github.com/webdriverio/webdriverio/blob/736113e28a369e66898a1d372fc295e6e2726736/packages/wdio-allure-reporter/src/index.ts#L444